### PR TITLE
Improved the code coverage of plugin/pkg/admission/exec.

### PR DESCRIPTION
**Pull request by k82cn**
part of #39559 

refer to the attachment for code coverage: [combined-coverage.html.gz](https://github.com/kubernetes/kubernetes/files/746891/combined-coverage.html.gz)

### CODE REVIEW

**Comment by k82cn** [comment]
cc @timothysc , using `NewDenyEscalatingExec` will avoid modifying code but test passed.

File: plugin/pkg/admission/exec/admission_test.go [hunk 1]
```diff
@@ -28,6 +28,15 @@ import (
 	"k8s.io/kubernetes/pkg/genericapiserver/registry/rest"
 )
 
+func newAllowEscalatingExec() admission.Interface {
```
Comment (timothysc) [inline plugin/pkg/admission/exec/admission_test.go]: generally lgtm, but I'm not familiar with why this param combination == Allow.  
Might be worthy of a comment imo.

**Review by timothysc — APPROVED** [review]
Only comment request b/c I'm not intimately familiar with this section of code.

**Comment by k82cn** [comment]
@derekwaynecarr , any comments for that?

**Comment by k82cn** [comment]
@k8s-bot unit test this

**Comment by k82cn** [comment]
@derekwaynecarr , ping for more comments :).

**Comment by timothysc** [comment]
lgtm, @derekwaynecarr ping...

**Comment by k82cn** [comment]
@derekwaynecarr , ping :).

**Comment by k82cn** [comment]
Unit test is failed with latest codes, not sure what happened. In my backlog, will correct it later.

```
I0310 12:01:03.088] FAIL	k8s.io/kubernetes/plugin/pkg/admission/exec [build failed]
```

**Comment by k82cn** [comment]
Interface was changed, updated accordingly; no logic change.

**Pull request merged**